### PR TITLE
feat: Tabs component migrated to design system

### LINKED ISSUE #17320: [Task]: Migrate tabs component to design system
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

_No response_

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

Migrated tabs component migrated to design system and changed the imports.

Fixes #17320

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested in local

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/components/editorComponents/form/FormDialogComponent.tsx [hunk 1]
```diff
@@ -3,8 +3,7 @@ import { isPermitted } from "pages/Applications/permissionHelpers";
 import Dialog from "components/ads/DialogComponent";
 import { useDispatch } from "react-redux";
 import { setShowAppInviteUsersDialog } from "actions/applicationActions";
-import { TabComponent, TabProp } from "components/ads/Tabs";
-import { Text, TextType, IconName } from "design-system";
+import { Text, TextType, IconName, TabComponent, TabProp } from "design-system";
```
Comment (tanvibhakta) [inline app/client/src/components/editorComponents/form/FormDialogComponent.tsx:6]:
```suggestion
import { IconName, TabComponent, TabProp, Text, TextType } from "design-system";
```

File: app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useMemo, useState } from "react";
 import styled from "styled-components";
 
 import { Colors } from "constants/Colors";
-import { TabTitle, TabComponent, TabProp } from "components/ads/Tabs";
+import { TabTitle, TabComponent, TabProp } from "design-system";
```
Comment (tanvibhakta) [inline app/client/src/pages/Editor/PropertyPane/PropertyPaneTab.tsx:5]:
```suggestion
import { TabComponent, TabProp, TabTitle } from "design-system";
```

**Review by tanvibhakta — APPROVED** [review]

**Review by tauantcamargo — DISMISSED** [review]

**Review by tanvibhakta — APPROVED** [review]

**Comment by shadabbuchh** [comment]
This PR has been tested. No issues found.

**Pull request merged**